scripts/Diffraction/isis_powder/routines/focus.py

```diff
@@ -136,8 +141,9 @@ def _divide_one_spectrum_by_spline(spectrum, spline, instrument):
 
 
 def _divide_by_vanadium_splines(spectra_list, vanadium_splines, instrument):
-    if hasattr(vanadium_splines, "OutputWorkspace"):  # vanadium_splines is a group
+    if hasattr(vanadium_splines, "OutputWorkspace"):
         vanadium_splines = vanadium_splines.OutputWorkspace
+    if type(vanadium_splines)is WorkspaceGroup:  # vanadium splines is a workspacegroup
```

Comment:
Missing space between `type(vanadium_splines)` and `is`.

Reply:
Fixed